src/contact/contact.service.ts

```diff
@@ -0,0 +1,50 @@
+import { v4 as uuidv4 } from 'uuid';
+import { addedData201, internalServerError500, OK200 } from '../constants';
+import { WorkWithDB } from '../interfaces';
+import { sanitizeRawArray } from '../utils/utils';
+import { GetContactFromDB, IContact } from './contact.interfaces';
+import { Contact } from './contact.model';
+
+export const createNewContact: WorkWithDB = async (req) => {
+  const { name, email } = JSON.parse(req.body.contact) as IContact;
+  try {
+    const res = await Contact.create({
+      uuid: uuidv4(),
+      name,
+      email,
+    });
+    return addedData201;
+  } catch (err) {
+    return internalServerError500;
+  }
+};
+
+export const updateExistedContact: WorkWithDB = async (req) => {
+  const uuid = req.params.contactId;
+  const { name, email } = JSON.parse(req.body.contact) as { name: string; email: string };
+  try {
+    const res = await Contact.update(
+      {
+        name,
+        email,
+      },
+      {
+        where: {
+          uuid,
+        },
+      }
+    );
+    return OK200;
+  } catch (err) {
```

Comment:
Why you expect error parameter if you don't use it in the response?

Reply:
changed